sequencer/src/block.rs

```diff
@@ -190,6 +202,19 @@ impl Header {
             transactions_root,
             fee_merkle_tree_root,
             block_merkle_tree_root,
+            builder_address: None,
+            builder_signature: None,
+            builder_fee_amount: None,
+        };
+
+        let hash =
+            ethers::utils::hash_message(serde_json::to_string(&header).unwrap().into_bytes());
```

Comment:
I think it's better to use the commitment scheme we use everywhere else to hash the header (so just `header.commit()`. Then you can use the low-level [`sign`](https://docs.rs/ethers/latest/ethers/core/k256/ecdsa/signature/trait.Signer.html#method.sign) method to sign the commitment (which is just a byte vector)